bedrock/mozorg/templates/mozorg/internet-health/index.html

```diff
@@ -96,9 +98,11 @@ <h3 class="card-heading">{{ _('Let your voice be heard… literally') }}</h3>
             {% endtrans %}
             </p>
 
-            <a href="https://voice.mozilla.org/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=ih-content-hub" class="cta-link" data-link-name="Record in Common Voice" data-link-type="link" data-link-position="common-voice">
-              {{ _('Record in Common Voice') }}
-            </a>
+            {% trans url='https://voice.mozilla.cz/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=ih-content-hub' if LANG == 'cs' else 'https://voice.mozilla.org/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=ih-content-hub' %}
+              <a href="{{ url }}" class="cta-link" data-link-name="Record in Common Voice" data-link-type="link" data-link-position="common-voice">
+                {{ _('Record in Common Voice') }}
```

Comment:
You're also wrapping a string inside another trans block here, which isn't valid. Please test your changes 👍